0051-helium-dao.md

```diff
@@ -1,325 +1,215 @@
-distributes the determined amount of HNT to a **DNP subDAO multi-signature
-wallet**, the addresses of which comprise the set of Validators of that DNP.
+Every epoch, DNP Oracles calculate each miners’ rewards based on data transferred in State Channels and distributes it to accounts on the L1 according to the emissions schedule for the subDAO.
 
-*DNT Issuance*
+*Proof-of-Coverage Process*
 
-Each DNP DAO then **submits the entire amount of HNT into its Programmatic Treasury** in
-order to mint new DNT for Hotspots and Validators participating in data transfer
-and Proof-of-Coverage.
+The DNP subDAO defines the rules for Proof-of-Coverage activity. In a given epoch, Challengers, Challenges, and Witnesses ****(in the case of a three party POC process) ****relay the results of challenges to DNP Oracles.
 
-In a given epoch, if the DNP Hotspots were to receive some amount of HNT from
-the Helium DAO, the DNP subDAO multi-signature wallet distributes a
-**corresponding amount of newly minted DNT of equal market value** as per the
-Programmatic Treasury to the Hotspots in its network based on its mining rules.
+Every epoch, DNP Oracles calculate each miners’ rewards based on proof of coverage data and distribute DNT to accounts on the L1 according to the emissions schedule for the subDAO.
 
-*Note that all new issuance in the DNP Programmatic Treasury through HNT emissions are
-distributed to Hotspots and Validators.*
+*HNT Emissions Calculation to subDAO Treasury Reserve*
 
-*DNT Issuance*
+At the end of a given epoch, the HNT emissions contract at the Helium DAO L1 calculates the amount of HNT to reward each DNP as per the DAO utility score in that given epoch. The emissions contract subsequently distributes the determined amount of HNT to the DNP subDAO treasury which is controlled by the DNP oracles.
 
-If a given address wishes to redeem their DNT, they can **“sell” against the
-Programmatic Treasury** to receive the market rate of underlying HNT.
+*Oracle Process*
 
-The transaction involves sending the required amount of DNT tokens to a **burn
-address**, and receiving the determined amount of HNT tokens from the **DNP
-subDAO multi-signature wallet**.
+The DNP subDAO defines the rules for oracles to come to consensus, and distributes a portion of DNT emissions in a given epoch for DNP oracles.
 
-The address **relays the message of the sale to the set of DNP Validators**,
-which then adjust the position of the Programmatic Treasury in the following epoch after
-adjusting for all other sales and any new issuance.
+![https://lh6.googleusercontent.com/6boBre9RQbrytVAu-kKZgqimQ-d5n19SUbw8IOkJDnlaazep_4dTn2yRHU_TGw_zYtbizMYWUY5k8ODeO7MiPNgXslJcOygxBICiVdB4NpzxstK0Cb5wuaK1LEpLsLyj68bWe1KIfkmiOz7ejQ](https://lh6.googleusercontent.com/6boBre9RQbrytVAu-kKZgqimQ-d5n19SUbw8IOkJDnlaazep_4dTn2yRHU_TGw_zYtbizMYWUY5k8ODeO7MiPNgXslJcOygxBICiVdB4NpzxstK0Cb5wuaK1LEpLsLyj68bWe1KIfkmiOz7ejQ)
```

Comment:
I think this diagram is a little confusing. specifically the transfer between end user and hotspot.